wye/profiles/views.py

```diff
@@ -97,3 +100,38 @@ def dispatch(self, *args, **kwargs):
             return super(ProfileEditView, self).dispatch(*args, **kwargs)
         else:
             raise PermissionDenied
+
+
+class ContactFormView(FormView):
+    form_class = ContactUsForm
+    template_name = 'contact.html'
+    success_url = '/thankyou'
+
+    def form_valid(self, form):
+
+        email_context = Context({
+            'contact_name': form.cleaned_data['name'],
+            'contact_email': form.cleaned_data['email'],
+            'comments': form.cleaned_data['comments'],
+            'mobile': form.cleaned_data['mobile_number'],
+            'query_topic': form.cleaned_data['query_topic']
+            })
+
+        subject = "PythonExpress New Contact Form Submission by %s" % (form.cleaned_data['name'])
```

Comment:
It was `PythonExpress Feedback by and` so here it will be `"PythonExpress Feedback by %s " % (form.cleaned_data['name'] `. Right?